### PR TITLE
Source Hubspot: revert to 1.2.0 

### PR DESCRIPTION
**Pull request by clnoll**
https://github.com/airbytehq/oncall/…issues/2677

### CODE REVIEW

**Review by girarda — APPROVED** [review]

File: airbyte-integrations/connectors/source-hubspot/metadata.yaml [hunk 1]
```diff
@@ -14,6 +14,7 @@ data:
   registries:
     cloud:
       enabled: true
+      dockerImageTag: 1.2.0
```
Comment (brianjlai) [inline airbyte-integrations/connectors/source-hubspot/metadata.yaml:17]: revert the dockerImageTag above as well if we know there is a bug in the latest release?

Comment (brianjlai) [reply airbyte-integrations/connectors/source-hubspot/metadata.yaml:17]: ah sorry nvm i got wired crossed in that I thought we were also going to revert the entire PR back

**Review by brianjlai — APPROVED** [review]

**Comment by octavia-approvington** [comment]
Approve this <br/> imagine the commander saying yes

**Review by octavia-approvington — APPROVED** [review]

File: airbyte-integrations/connectors/source-hubspot/metadata.yaml (hunk 1 above)
Comment (clnoll) [reply airbyte-integrations/connectors/source-hubspot/metadata.yaml:17]: Dangit, didn't see this. I'll follow up with Support to see if we should do that.

**Comment by octavia-squidington-iii** [comment]
##  source-hubspot test report (commit [`a4feace7c2`](https://github.com/airbytehq/airbyte/pull/29187/commits/a4feace7c2b65d672d725afeb1621c6c228a819c)) - ❌

⏲️  Total pipeline duration: 11mn18s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-hubspot/metadata.yaml | ✅        |
| Connector version semver check                                        | ✅        |
| QA checks                                                             | ✅        |
| Code format checks                                                    | ✅        |
| Connector package install                                             | ✅        |
| Build source-hubspot docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                            | ✅        |
| Integration tests                                                     | ✅        |
| Acceptance tests                                                      | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/p0-hubspot-authentication-errors-8-7-23/1691441225/a4feace7c2b65d672d725afeb1621c6c228a819c/source-hubspot/1.3.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:a4feace7c2b65d672d725afeb1621c6c228a819c)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-hubspot test
```

**Pull request merged**